app/code/Magento/Catalog/Model/Product/Image/Cache.php

```diff
@@ -127,4 +154,93 @@ protected function processImageData(Product $product, array $imageData, $file)
 
         return $this;
     }
+
+    /**
+     * Get themes in use
+     *
+     * @return array
+     */
+    public function getThemesInUse()
```

Comment:
Does this method need to be public?
Please consider extracting it into appropriate service, as `getThemesInUse` does not seem to have too much in common with Product Image Cache Model.

Reply:
Where would you suggest I should put it? Inside themes?